public/app/features/plugins/admin/components/GetStartedWithPlugin/GetStartedWithApp.tsx

```diff
@@ -19,9 +19,8 @@ export function GetStartedWithApp({ plugin }: Props): React.ReactElement | null
   if (!pluginConfig) {
     return null;
   }
-
   // Enforce RBAC
-  if (!contextSrv.hasPermissionInMetadata(AccessControlAction.PluginsWrite, plugin)) {
+  if (!contextSrv.hasPermission(AccessControlAction.PluginsWrite)) {
```

Comment:
what is the difference in these two methods?

Reply:
`hasPermissionInMetadata` checks if the object has specific access in our case plugin should contain PluginWrite access policy
`hasPermission` checks if the current user has a permission, in our case user you are logged in has PluginsWrite permission

Maybe plugin object was changed or smth else changed, but now plugin does not have any of accessControl in it, that's why I have changed the method